src/bitwarden_workflow_linter/rules/run_actionlint.py

```diff
@@ -0,0 +1,108 @@
+"""A Rule to run actionlint on workflows."""
+
+from typing import Optional, Tuple
+import subprocess
+import platform
+import urllib.request
+import os
+
+from ..rule import Rule
+from ..models.workflow import Workflow
+from ..utils import LintLevels, Settings
+
+
+def install_actionlint(platform_system: str) -> Tuple[bool, str]:
+    """If actionlint is not installed, detects OS platform
+    and installs actionlint"""
+
+    error = f"An error occurred when installing Actionlint on {platform_system}"
+
+    if platform_system.startswith("Linux"):
+        return install_actionlint_source(error)
+    elif platform_system == "Darwin":
+        try:
+            subprocess.run(["brew", "install", "actionlint"], check=True)
+            return True, ""
+        except (FileNotFoundError, subprocess.CalledProcessError):
+            return False, f"{error} : check Brew installation"
+    elif platform_system.startswith("Win"):
+        try:
+            subprocess.run(["choco", "install", "actionlint", "-y"], check=True)
+            return True, ""
+        except (FileNotFoundError, subprocess.CalledProcessError):
+            return False, f"{error} : check Choco installation"
+    return False, error
+
+
+def install_actionlint_source(error) -> Tuple[bool, str]:
+    """Install Actionlint Binary from provided script"""
+    url = "https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash"
+    version = "1.6.17"
```

Comment:
❓ Is this the best place to store the version of actionlint today?

Reply:
Also, is it possible to use Renovate to automate the actionlint version updates here?

Reply:
I am not sure on renovate, since this might require a custom handler for renovate. However, moving this version into something like https://github.com/bitwarden/workflow-linter/blob/main/src/bitwarden_workflow_linter/default_settings.yaml would make it much easier to maintain.

Reply:
@michalchecinski I don't know how renovate works outside of package and action management. @mimartin12 I'd love to have it in a more centralized place where we can track versions but I don't feel like it fits in with settings or package versions. Do you have a suggestion?

Reply:
Looking at the code right now, settings was the only place that stood out. If there is no way to set settings for each rule, then that sounds like a feature to me and is outside of scope IMO.

Reply:
I think it's definitely possible to add actionlint version to the settings file, but I think creating that access is out of scope unless someone with more knowledge wanted to take it on.